frontend/mr-admin-flate/src/pages/avtaler/AvtaleDetaljer.tsx

```diff
@@ -219,8 +219,8 @@ export function AvtaleDetaljer() {
           <Metadata
             header={avtaletekster.tiltaksarrangorHovedenhetLabel}
             verdi={
-              <Link to={`/arrangorer/${arrangor.id}`}>
-                {arrangor.navn} - {arrangor.organisasjonsnummer}
+              <Link to={`/arrangorer/${arrangor?.id}`}>
+                {arrangor?.navn} - {arrangor?.organisasjonsnummer}
```

Comment:
Kunne vi gjort en null-sjekk i forkant av hele denne blokken, i stedet for å nullish-coalescing hver gang referer til arrangør? Nå kan det se ut som vi enter opp med tomme verdier ` - ` og lenker til `/arrangorer/undefined` i tilfellet der arrangør mangler.

Kanskje det også hadde vært nyttig å legge inn en info-melding om at arrangør mangler også? 

F.eks. noe ala

```ts
arrangor ? (
  <VStack gap="5">
    <Metadata
      header={avtaletekster.tiltaksarrangorHovedenhetLabel}
      verdi={
        <Link to={`/arrangorer/${arrangor.id}`}>
          {arrangor.navn} - {arrangor.organisasjonsnummer}
        </Link>
      }
     />
   
     osv ...

  </Vstack>
) : (
  <AvtaleErKladdOgArrangørManglerMelding />
)
```